src/Client/parseConnectionString.ts

```diff
@@ -285,8 +287,9 @@ const verifyKeyValuePair = (
 
       return { key, value: parsedValue };
     }
+    case "connectionName":
     case "tlsCAFile": {
-      return { key, value };
+      return { key, value: decodeURIComponent(value) };
```

Comment:
**QUESTION:** Does that mean that from now on, users will need to encode that part of the connection string?

Reply:
No, it just means they can if they want to, reducing limitations. 

So everything would work as before: 
```
decodeURIComponent('/path/to/file.ca')
'/path/to/file.ca'
```

But before this change if we had a connection string like: 
`?tlsCAFile=path/to/george & oskar's badly named certificate file.ca&throwOnAppendError=true`

It would (and still would) fail to parse (as its not a valid uri), giving something like `tlsCAFile`: `path/to/george`, then it would throw an error. 

Now you have the option to encode the chars:

`?tlsCAFile=path%2Fto%2Fgeorge%20%26%20oskar's%20badly%20named%20certificate%20file.ca&throwOnAppendError=true`

and it would parse it out correctly.

Reply:
Thanks for an explanation 👍